website/package.json

```diff
@@ -97,4 +97,4 @@
   "lint-staged": {
     "*.{js,ts,jsx,tsx,md}": "npm run lint"
   }
-}
+}
```

Comment:
@wendyvasq Just a piece of advice Connor gave me, to avoid weird merge conflicts that can occur at the end of file -- there's a setting in VS Code to add a new line at the end of the file. I think most of us having this turned on.

No need to fix this in this PR; just something I thought you should know!

<img width="538" alt="Screenshot 2023-03-31 at 9 20 34 AM" src="https://user-images.githubusercontent.com/72401230/229131270-ffbdb5d9-abcf-491d-b883-dde636a48142.png">

Reply:
@lfracalossiFA I had no idea! I'll also turn this on!